public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -70,11 +71,8 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
 
     return (
       <>
-        <div
-          className="gf-form-inline gf-form-inline--xs-view-flex-column flex-grow-1"
-          data-testid={this.props['data-testid']}
-        >
-          <div className="gf-form--grow flex-shrink-1 min-width-15">
+        <InlineFieldRow data-testid={this.props['data-testid']}>
```

Comment:
This change is breaking the query editor when you resize it.

![imagen](https://github.com/grafana/grafana/assets/1069378/03633980-af29-4858-aaa4-8c64f90a7178)

![imagen](https://github.com/grafana/grafana/assets/1069378/8330c638-91d5-43dd-88cd-8cfd10136e19)